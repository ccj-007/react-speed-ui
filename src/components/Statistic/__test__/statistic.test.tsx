import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Statistic } from '../../index';

describe('test Statistic component', () => {
  it('should render', () => {
    const { asFragment } = render(<Statistic />);
    expect(asFragment()).toMatchSnapshot();
  });
});
