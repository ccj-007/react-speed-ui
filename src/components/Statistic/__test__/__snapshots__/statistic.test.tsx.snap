// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Statistic component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-statistic"
  >
    <div
      class="speed-container speed-container-column speed-container-between"
    >
      <div
        class="speed-statistic-title"
      />
      <div
        class="speed-statistic-value"
      >
        undefined
      </div>
    </div>
  </div>
</DocumentFragment>
`;
